meta/src/lib.rs

```diff
@@ -67,99 +67,3 @@ pub fn parse_and_optimize(
 
     Ok((defaults, optimizer::optimize(ast)))
 }
-
-#[doc(hidden)]
-pub static UNICODE_PROPERTY_NAMES: &[&str] = &[
```

Comment:
I know it the breaking change 😂

Because it a static const, it's hard to assign values from `pest::unicode::unicode_property_names`.

I will try to do that today.

Reply:
@tomtau https://github.com/pest-parser/pest/pull/751/commits/3b6d1dc4ed0be02bca32782b2bc68dd6d5efbe0d

I have updated that.